frontend/src/app/components/contact-us/contact-us.component.html

```diff
@@ -20,7 +20,6 @@ <h2>Contact Us</h2>
          <!--These buttons will allow users to log in or sign up, the [disabled] code state that the signInForm (that we set above)
          needs to be completed before the user can press either of the buttons-->      
          <button type="submit" 
-                 (click)="submit()"
```

Comment:
Not sure about this "Contact-us page"... I just removed/disabled some code from it since I didn't bother making Routes/Controller for it

Reply:
That's fine, we'll probably revisit the contact page later and bring it up to standard, since that other person made it I believe ya?